packages/generators/app/lib/utils/check-requirements.js

```diff
@@ -17,7 +17,7 @@ module.exports = function checkBeforeInstall() {
   }
 
   // warn if not using a LTS version
-  else if (semver.satisfies(currentNodeVersion, '15.x.x || 17.x.x || 19.x.x')) {
+  else if (semver.satisfies(currentNodeVersion, '14.x.x' || '15.x.x' || '17.x.x || 19.x.x')) {
```

Comment:
I'm not sure we should consider `14.x.x` as 'non-LTS', as we do not do it for version 12 for example.
Maybe we can also add `21.x.x`, wdyt ?